alethzero/plugins/browser/Cors.ui

```diff
@@ -1,119 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
  <class>Cors</class>
- <widget class="QDialog" name="Cors">
+ <widget class="QWidget" name="Cors">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>534</width>
```

Comment:
If this is a pixel value, then this will be another dialog that does not display correctly on my screen ;-)